App/Redux/ProcessingImagesRedux.ts

```diff
@@ -80,14 +86,17 @@ export interface ProcessingImage {
 }
 
 export interface ProcessingImagesState {
-  readonly images: ProcessingImage[]
+  readonly images: ProcessingImage[],
+  readonly lastPhotoRefresh: number // epoch in seconds
```

Comment:
This is that date we need to set. We can update it when the option is toggled and whenever we run the query.

Reply:
oop, also changed this to milliseconds. just kept everything in JS as milliseconds until the moment it makes the objc request in seconds